src/transformers/models/bert/modeling_bert.py

```diff
@@ -504,9 +547,13 @@ def custom_forward(*inputs):
                     layer_head_mask,
```

Comment:
Also, while we're at it -> maybe switch to only keyword args here? It's usually safer this way